apps/shell/bin/setup

```diff
@@ -50,17 +93,19 @@ chdir APP_ROOT do
   end
 
   puts "\n== Building Shell App =="
-  puts "DEFAULT_SSHHOST = #{DEFAULT_SSHHOST || OLD_SSHHOST || "localhost"}"
+  puts "DEFAULT_SSHHOST = #{DEFAULT_SSHHOST || OLD_SSHHOST}"
```

Comment:
I'd just get rid of this line that outputs default ssh host. This was added a long time ago when we didn't have build by rpm and no configuration in /etc/